docs/source/upgrade/example/coinV1/daml.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
```

Comment:
Can we get a test that builds this using `daml build` as separate projects? Maybe just a simple `sh_test` that first runs `damlc build` in the `coinV1` directory and then `damlc build` in the `coinV2` directory.

Reply:
Sure. 
Can you point me to an doc example where we do this, so I can crib from the setup!

Reply:
Not sure what the best example is but https://github.com/digital-asset/daml/blob/master/compiler/damlc/tests/BUILD.bazel#L380 is hopefully good enough.

Reply:
done